src/snippets/reference/nodejs/IPLocation.js

```diff
@@ -13,6 +13,11 @@ const aj = arcjet({
 const server = http.createServer(async function (req, res) {
   const decision = await aj.protect(req);
 
+  if (decision.isDenied()) {
+    res.writeHead(403, { "Content-Type": "application/json" });
+    res.end(JSON.stringify({ error: "Forbidden", reason: decision.reason }));
```

Comment:
Needs an early return
```suggestion
    res.end(JSON.stringify({ error: "Forbidden", reason: decision.reason }));
    return;
```